internal/api/core/routes.go

```diff
@@ -9,6 +9,10 @@ import (
 
 var CURRENT_PREFIX string = fmt.Sprintf("/api/v%02d", util.MustGetAPIVersion())
 
+// API routes will be empty until RunServer() is called.
+// This is a cached version of api.GetRoutes() for packages that cannot access `internal/api` due to import cycles.
```

Comment:
Swap the order of these sentences.
You always want your first sentence to be the general description/overview.